Modules/Core/Transform/include/itkTransformGeometryImageFilter.h

```diff
@@ -26,25 +26,28 @@
 namespace itk
 {
 /** \class TransformGeometryImageFilter
- * \brief Pseudo-Resample an image by modifying the geometric meta-data and not the pixels.
- *
- * The TransformGeometryImageFilter will generate a physical memory-modified version of
- * the input image if the input transform is not identity. Its neglectful use
- * can be a source of problems: e.g. it can exhaust the memory if the image is
- * very large, and it WILL reduce the image quality when there are lots of
- * transforms to be superimposed for the input image. Often times, we are not
- * interested in the intermediate transformed images.
- *
- * If all the transforms are rigid, there is a far superior way to achieve a similar result.
- * Updating image metadata in-place removes the accumulated resampling errors
- * as well as eliminating the expense of accessing the physical memory of the image.
- * We need to compose all the transforms beforehand to make use of this filter.
- *
- * \param \c RigidTransform Currently must be a VersorRigid3DTransform.
+ * \brief Modify an image's geometric meta-data, changing its "physical" extent.
+ *
+ * The TransformGeometryImageFilter "physically" changes the image in
+ * space using the inverse of the given transformation. The
```

Comment:
Saying "inverse of the given transformation" will be confusing to most users. The high level view is that it performs the same transformation as the resample image filter, where each  pixels physical location ti transformed by provided transform. Please say what the filter does not what it does not not do.

Reply:
done